src/components/ToastContainer.tsx

```diff
@@ -0,0 +1,51 @@
+import { ToastContainer as ToastifyContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { styled } from 'stitches.config';
+
+const StyledContainer = styled(ToastifyContainer, {
+  '.Toastify__toast': {
+    background: '$formInputBackground',
+    borderRadius: '9px',
+    border: '1px solid $formInputBackground',
+    minWidth: '400px',
+    maxWidth: '640px',
+    width: '80%',
+  },
+  '.Toastify__toast-icon': {
+    marginInlineEnd: '20px',
+  },
+  '.Toastify__progress-bar': {
+    height: '1px',
+  },
+  '.Toastify__toast--info': {},
+  '.Toastify__toast--success': {
+    borderColor: '$successColor',
+  },
+  '.Toastify__toast--error': {
+    borderColor: '$errorColor',
+  },
+  '.Toastify__toast--success *': {
+    color: 'var(--toastify-text-color-success)',
```

Comment:
@wingfeatherwave are there fixed color variable names we can just use for these?

Reply:
I think using the toastify values is the right thing to do since success and error color isn't different between dark/light.  BTW, I believe you can write this as `$toastify-text-color-success` instead of `var(--toastify-text-color-success)`

Reply:
I just tried to switch to $ vars and it isn't working...

<img width="613" alt="Screenshot 2023-01-12 at 10 14 10 AM" src="https://user-images.githubusercontent.com/83605543/212146457-953c1c10-fc99-49e9-9c29-cb05106d64b2.png">

I think these are used by CSS itself not Stitches, so they need to be in the plain old CSS syntax... or I'm doing something else wrong.